docs/2019-03-28-All-Path-Reachability-Proofs.md

```diff
@@ -0,0 +1,240 @@
+Proving All-Path Reachability Claims
+====================================
+
+This document details All-Path Reachability without solving the
+most-general-unifier (MGU) problem.
+MGU will be detailed in a separate document.
+
+Background
+----------
+
+Recommended reading: [proving One-Path Reachability
+claims](2018-11-08-One-Path-Reachability-Proofs.md).
+
+
+Definitions
+-----------
+
+### Weak always
+
+Given a formula `φ`, let `[w]φ` denote the formula “weak always” `φ`,
```

Comment:
weak always  = all path eventually?

Reply:
essentially, yes.  This is how @hreada named it.